buildscripts/CMakeLists.txt

```diff
@@ -348,6 +352,21 @@ ExternalProject_Add(mygui
 	INSTALL_COMMAND ${wrapper_command} $(MAKE) install
 )
 
+ExternalProject_Add(lz4
+		URL https://github.com/lz4/lz4/archive/v${LZ4_VERSION}.tar.gz
+		https://github.com/xyzz/openmw-deps/releases/download/v0/v${LZ4_VERSION}.tar.gz
+		URL_HASH ${LZ4_HASH}
+		DOWNLOAD_DIR ${download_dir}
+
+		CONFIGURE_COMMAND ${wrapper_command} cmake <SOURCE_DIR>/contrib/cmake_unofficial/
+		${COMMON_CMAKE_ARGS}
+		-DBUILD_STATIC_LIBS=ON
+
+		BUILD_COMMAND ${wrapper_command} $(MAKE)
+
+		INSTALL_COMMAND ${wrapper_command} $(MAKE) install
+		)
```

Comment:
some formatting issues here, use a single tab for inner blocks, don't indent the last )